doc/user_guide.rst

```diff
@@ -6,69 +6,107 @@ User Guide
 ==========
 
 
-Basic setup
+Basic usage
 ===========
 
-Basic mathematical operations involving numbers with uncertainties
-only require a simple import:
+Basic mathematical operations involving numbers with uncertainties requires
+importing the :func:`ufloat` function which creates numbers with uncertainties:
 
 >>> from uncertainties import ufloat
 
-The :func:`ufloat` function creates numbers with uncertainties. Existing
-calculation code can usually run with no or little modification and
-automatically produce results with uncertainties.
+The :mod:`uncertainties` module contains sub-modules for :ref:`advanced
+mathematical functions <advanced math operations>`, and :doc:`arrays and
+matrices <numpy_guide>`, which can be accessed with
 
-.. The "import uncertainties" is put here because some examples requires
-   uncertainties to have been imported (and not only ufloat).
+>>> import uncertainties as un
```

Comment:
does this conflict with any other packages?

Reply:
dunno.  One could probably ask that about any abbreviation.